typing/types.ml

```diff
@@ -403,6 +404,7 @@ and ext_status =
     Text_first                     (* first constructor of an extension *)
   | Text_next                      (* not first constructor of an extension *)
   | Text_exception                 (* an exception *)
+  | Text_effect
```

Comment:
Same as `Oext_effect`, this kind of extension constructor declaration doesn't exist anymore.

Reply:
Fixed in https://github.com/ocaml/ocaml/pull/12309/commits/0479cffb3b133c0084683235b231daa02d6bdb6b.